README.md

```diff
@@ -100,6 +100,11 @@ stack build
 cd staging
 spago install
 
+# Disable glob expansion to prevent 'ModuleNotFound' errors.
+# `purs` will handle that for us. See #220 for more contxt.
+#    Note: to re-enable, run `set +o noglob`
+set -o noglob
```

Comment:
I think if you do `(set -o noglob && stack exec …)` that ought to still work and won’t affect the current shell, so you won’t need to re-enable globbing later.

Reply:
A common alternative is also to `set +o noglob` after the `stack exec` command

Reply:
Right - what I’m saying is that I think it’s preferable not to have to do that. It’s annoying to have to turn it on and off if you’re running normal commands which do rely on globbing in between running the Try PureScript server

Reply:
Sorry nvm, I thought this was a script, but it's dev instructions

Reply:
Done.